src/sage/rings/function_field/order_rational.py

```diff
@@ -427,6 +438,8 @@ class FunctionFieldMaximalOrderInfinite_rational(FunctionFieldMaximalOrderInfini
     """
     Maximal infinite orders of rational function fields.
 
+    These are not finitely-generated rings in general.
+
     INPUT:
```

Comment:
You mean it as K-algebra where K is the base field?

"in general"? Isn't it so for all? 

I wonder what are the (canonical) generators...

Yes, definitely, `gen()` does not make sense for the infinite valuation ring.

Reply:
It is not even finitely generated over K[1/x], right ?

I can remove the added sentence if you prefer.

Reply:
> It is not even finitely generated over K[1/x], right ?

It seems so. I don't know for sure.
 
> I can remove the added sentence if you prefer.

Yes.